src/Options/FixtureOptions.php

```diff
@@ -343,6 +341,27 @@ public function getCoreResolved(): string {
     return $version;
   }
 
+  /**
+   * Check if core version resolves to the constraint provided.
```

Comment:
```suggestion
   * Checks if core version resolves to the constraints provided.
```

I don't make the rules. 🤷 😉

> For most functions... summary lines must start with a third person singular present tense verb, and they must explain what the function does. Example: "Calculates the maximum weight for a list."

\- https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#functions

Reply:
Awesome :)